portfolio/src/main/webapp/script.js

```diff
@@ -40,4 +40,33 @@ function KaylaTrivia(){
     else{
          alert(`Your final score is ${points}  Oof maybe try again?`);
     }
+}
+function getMessage() {
+  console.log('Fetching a random quote.');
```

Comment:
While printing to the console is useful for development/debugging, these lines generally don't make it into the code. Feel free to use locally but I'd recommend not committing it with the code (here and elsewhere below).